tests/fixtures/blast/blastSettingsConfig.json

```diff
@@ -590,16 +594,27 @@
     }
   },
   "database_sequence_types": {
+    "dna_sm": "dna",
     "dna": "dna",
     "cdna": "dna",
     "pep": "protein"
   },
   "programs_configurator": [
+    {
+      "sequence_type": "dna_sm",
```

Comment:
This (and same in another entry below) is probably wrong. Look at the configurator for protein below — it says `sequence_type: "protein"`, rather that `sequence_type: "pep"`.

I actually suspect that these two entries that you added to the configurator are unnecessary. Need to check and update Andres's config.

Reply:
Doh! right, this is sequence type and not database type!